frontend-v2/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,147 @@
+"use client";
+
+import { useState } from "react";
+import Image from "next/image";
+import logo from "../../../public/uni-lectives.svg";
+import { BookOpenIcon, PencilSquareIcon, ShieldCheckIcon, UserCircleIcon, BarsArrowDownIcon, BarsArrowUpIcon, MoonIcon, ArrowRightOnRectangleIcon } from "@heroicons/react/24/outline";
+import Tooltip from "@/components/Tooltip/Tooltip";
+
+export default function Navbar() {
+  const [doLogout, setDoLogout] = useState(false);
+  const [collapsed, setCollapsed] = useState(true);
+
+  return (
+    <>
+      {/* Collapsed View */}
+      {collapsed ? (
+        <div className="fixed flex flex-col items-center w-20 h-screen gap-4 p-4 bg-gray-50 z-50 xs:p-2 xs:w-15 xs:gap-2">
+          {/* Logo */}
+          <div className="flex items-center justify-between h-10 p-2">
+            <a href="/">
+              <Image
+                src={logo}
+                width={33}
+                height={33}
+                alt="logo"
+                priority
+              />
+            </a>
+          </div>
+          {/* Navbar Container */}
+          <div className="flex flex-col h-full w-full items-center justify-between border-t-2 border-gray-200">
+            {/* Top Section */}
+            <div className="flex flex-col gap-3 items-left py-3">
```

Comment:
can remove redundant flex col css if top section is inheriting styles from the navbar container